coins/src/adapters/tokenMapping.json

```diff
@@ -2938,6 +2962,21 @@
       "decimals": "18",
       "symbol": "ZBTC",
       "to": "coingecko#wrapped-bitcoin"
+    },
+    "0xe04d21d999faedf1e72ade6629e20a11a1ed14fa": {
+      "decimals": "18",
+      "symbol": "SolvBTC.m",
+      "to": "coingecko#solv-btc"
+    },
+    "0xb88a54ebbda8edbc1c2816ace1dc2b7c6715972d": {
+      "decimals": "18",
+      "symbol": "bcLP-stBTC-WBTC",
```

Comment:
is this bcLP from a Uni V2 fork? Can you explain how this is pegged to WBTC?

Reply:
This bcLP (bitCow LP) is a Stableswap LP (https://bitcow.gitbook.io/amm#stableswap-v1), representing the liquidity position of stBTC-WBTC on Bitcow. https://github.com/bitSmiley-protocol/bitcow/blob/main/contracts/BitcowLpToken.sol

About stBTC: stBTC is a rebasing token, user staked WBTC on Lorenzo for stBTC. https://docs.lorenzo-protocol.xyz/